dotcom-rendering/src/footballMatches.ts

```diff
@@ -441,3 +441,35 @@ const cleanTeamName = (teamName: string): string => {
 		.replace('Bialystock', 'Białystok')
 		.replace('Union Saint Gilloise', 'Union Saint-Gilloise');
 };
+
+// This comes from Frontend
+const paStatusToMatchStatus: Record<string, string> = {
+	KO: '1st', // The Match has started Kicked Off.
+	HT: 'HT', // The Referee has blown the whistle for Half Time.
+	SHS: '2nd', // The Second Half of the Match has Started.
+	FT: 'FT', // The Referee has blown the whistle for Full Time.
+	PTFT: 'FT', // Penalty Shootout Full Time.
+	Result: 'FT', // The Result is official.
+	ETFT: 'FT', // Extra Time, Full Time has been blown.
+	MC: 'FT', // Match has been Completed.
+	FTET: 'ET', // Full Time, Extra Time it to be played.
+	ETS: 'ET', // Extra Time has Started.
+	ETHT: 'ET', // Extra Time Half Time has been called.
+	ETSHS: 'ET', // Extra Time, Second Half has Started.
+	FTPT: 'PT', // Full Time, Penalties are To be played.
+	PT: 'PT', // Penalty Shootout has started.
+	ETFTPT: 'PT', // Extra Time, Full Time, Penalties are To be played.
+	Suspended: 'S', // Match has been Suspended.
+
+	// We don't really expect to see these the way we handle data in frontend
+	Resumed: 'R', // Match has been Resumed.
+	Abandoned: 'A', // Match has been Abandoned.
+	Fixture: 'F', // Created Fixture is available and had been Created by us.
+	'-': 'F', // this sneaky one is not in the docs
+	New: 'N', // Match A New Match has been added to our data.
+	Cancelled: 'C', // A Match has been Cancelled.
+};
+
+const replaceLiveMatchStatus = (status: string): string => {
+	return paStatusToMatchStatus[status] ?? status.slice(0, 2);
```

Comment:
In frontend we were logging when a status wasn't found in the mappings - but given that one hasn't been added in over a decade it seems unnecessary